src/templates/layout.hbs

```diff
@@ -7,14 +7,14 @@
 <html>
     <head>
         <meta charset="utf-8"/>
-        <meta property="Description" content="A free and open source Air Traffic Control Simulator."/>
-        <meta property="og:title" content="Openscope - Air Traffic Control Simulator"/>
-        <meta property="og:description" content="A free and open source Air Traffic Control Simulator."/>
+        <meta property="Description" content="Free and open source air traffic control simulator"/>
+        <meta property="og:title" content="openScope Air Traffic Control Simulator"/>
```

Comment:
`Air Traffic Control Simulator` is cased differently here, is that on purpose?

Reply:
I see what you're saying:
```
<meta property="Description" content="Free and open source air traffic control simulator"/>
<meta property="og:title" content="openScope Air Traffic Control Simulator"/>
<meta property="og:description" content="Free and open source air traffic control simulator"/>
```

They were cased based on which property they were under.
- I assume description to be somehow used to describe what the site is, so I used normal sentence-style capitalization.
- Title was cased like the title of a book

That was the thought process anwyay.

Reply:
https://github.com/openscope/openscope/pull/1024#discussion_r204261319

Responded to in the next comment section on `layout.hbs` below.